db_upgrade_scripts/mosip_ida/sql/1.2.1.0_to_1.3.0_rollback.sql

```diff
@@ -1,28 +1,38 @@
 -- ------------------------------------------------------------------------------------------
--- Rollback script for reverting Spring Batch version 5.0 migration as part of Java 21 migration.
+-- Rollback script for Reverting Spring Batch version upgrade changes.
 -- ------------------------------------------------------------------------------------------
 
--- Revert the changes to BATCH_STEP_EXECUTION
-ALTER TABLE BATCH_STEP_EXECUTION ALTER COLUMN START_TIME SET NOT NULL;
+-- Reverse the addition of the CREATE_TIME column
 ALTER TABLE BATCH_STEP_EXECUTION DROP COLUMN CREATE_TIME;
 
--- Revert the changes to BATCH_JOB_EXECUTION_PARAMS
+-- Reverse the modification of START_TIME column
+ALTER TABLE BATCH_STEP_EXECUTION ALTER COLUMN START_TIME SET NOT NULL;
+
+-- Reverse the removal of DATE_VAL, LONG_VAL, and DOUBLE_VAL columns
 ALTER TABLE BATCH_JOB_EXECUTION_PARAMS ADD COLUMN DATE_VAL TIMESTAMP;
 ALTER TABLE BATCH_JOB_EXECUTION_PARAMS ADD COLUMN LONG_VAL BIGINT;
 ALTER TABLE BATCH_JOB_EXECUTION_PARAMS ADD COLUMN DOUBLE_VAL DOUBLE PRECISION;
-ALTER TABLE BATCH_JOB_EXECUTION_PARAMS ALTER COLUMN PARAMETER_TYPE TYPE CHAR(6);
+
 ALTER TABLE BATCH_JOB_EXECUTION_PARAMS RENAME PARAMETER_TYPE TO TYPE_CD;
+
```

Comment:
TYPE_CD datatype is not reversed. We  should reverse the data type as well.

Reply:
corrected PR.